core/app/models/spree/order_taxation.rb

```diff
@@ -17,13 +17,15 @@ def initialize(order)
 
     # Apply taxes to the order.
     #
-    # This method will create or update adjustments on all line items and
-    # shipments in the order to reflect the appropriate taxes passed in. It
-    # will also remove any now inapplicable tax adjustments.
+    # This method will create or update adjustments on the order and all line
+    # items and shipments in the order to reflect the appropriate taxes passed
+    # in. It will also remove any now inapplicable tax adjustments.
     #
     # @param [Spree::Tax::OrderTax] taxes the taxes to apply to the order
     # @return [void]
     def apply(taxes)
+      update_adjustments(@order, taxes.order_taxes) if taxes.try(:order_taxes)
```

Comment:
Do we need to `try` on taxes? Two lines below, we're directly calling `taxes.line_item_taxes`, so it doesn't look like it could be `nil`.

Reply:
This was meant more of a safety check for method exists than a nil check. Meant to simplify: 
```ruby
taxes.respond_to?(:order_taxes) && taxes.order_taxes
```
I was just being overly cautious about the possibility of some apps overriding part of the tax system but still using the built-in `OrderTaxation` class. I'll remove the `#try`.

Reply:
Thanks, @adammathys!